Documentation/guides/k8s-aws-ansible/install.rst

```diff
@@ -0,0 +1,1430 @@
+    i-xxxxxxxxxxxxxxxxxxx   192.168.32.12   x.x.x.x ciliumMaster2   ami-xxxxxxxx    running us-west-1a  2017-06-16T06:08:24.000Z
+    i-xxxxxxxxxxxxxxxxxxx   192.168.32.13   x.x.x.x ciliumMaster3   ami-xxxxxxxx    running us-west-1a  2017-06-16T06:08:24.000Z
+    i-xxxxxxxxxxxxxxxxxxx   192.168.32.21   x.x.x.x ciliumSlave1    ami-xxxxxxxx    running us-west-1a  2017-06-16T06:08:24.000Z
+    i-xxxxxxxxxxxxxxxxxxx   192.168.32.22   x.x.x.x ciliumSlave2    ami-xxxxxxxx    running us-west-1a  2017-06-16T06:08:24.000Z
+    i-xxxxxxxxxxxxxxxxxxx   192.168.32.23   x.x.x.x ciliumSlave3    ami-xxxxxxxx    running us-west-1a  2017-06-16T06:08:24.000Z
+
+Backup current ones first
+
+::
+
+    10.3.0.11       mi1
+    10.3.0.12       mi2
+    10.3.0.13       mi3
+    10.3.0.21       si1
+    10.3.0.22       si2
+    10.3.0.23       si3
+
+
+    x.x.x.x     mp1
+    x.x.x.x     mp2
+    x.x.x.x     mp3
+    x.x.x.x     sp1
+    x.x.x.x     sp2
+    x.x.x.x     sp3
+
+| Update hosts file
+| $ sudo vi /etc/hosts
+
+::
+
+    192.168.32.11   mi1
+    192.168.32.12   mi2
+    192.168.32.13   mi3
+    192.168.32.21   si1
+    192.168.32.22   si2
+    192.168.32.23   si3
+
+    x.x.x.x     mp1 
+    x.x.x.x     mp2 
+    x.x.x.x     mp3 
+    x.x.x.x     sp1 
+    x.x.x.x     sp2 
+    x.x.x.x     sp3
+
+::
+
+    core@host:~/cilium/installation/caNew$
+    for host in mp1 mp2 mp3; do scp ca.pem ca-key.pem apiserver.pem apiserver-key.pem core@${host}:~/; done
+    scp ca.pem S1-worker.pem S1-worker-key.pem core@sp1:~/;
+    scp ca.pem S2-worker.pem S2-worker-key.pem core@sp2:~/;
+    scp ca.pem S3-worker.pem S3-worker-key.pem core@sp3:~/;
+
+3. Deploy Kubernetes Master Node(s)]
+-----------------------------------
+
+3.1 Set up ANSIBLE
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+::
+
+    core@host:~/cilium/installation$ mkdir ansible;cd ansible/;
+    core@host:~/cilium/installation/ansible$ vi hosts
+    [mps:children]
+    mp1
+    mp2
+    mp3
+    
+    [sps:children]
+    sp1
+    sp2
+    sp3
+    
+    [mis:children]
+    mi1
+    mi2
+    mi3
+    
+    [sis:children]
+    si1
+    si2
+    si3
+    
+    [mi1]
+    192.168.32.11
+    [mi2]       
+    192.168.32.12
+    [mi3]       
+    192.168.32.13
+    [si1]       
+    192.168.32.21
+    [si2]       
+    192.168.32.22
+    [si3]       
+    192.168.32.23   
+    
+    [mp1]
+    x.x.x.x
+    [mp2]     
+    x.x.x.x
+    [mp3]    
+    x.x.x.x
+    [sp1]    
+    x.x.x.x
+    [sp2]      
+    x.x.x.x
+    [sp3]     
+    x.x.x.x 
+    core@host:~/cilium/installation/ansible$ ansible-playbook -i hosts site.yml
+    
+    PLAY [mps,sps] **********************************************************************************************************************    ***************************************
+    
+    TASK [defunctzombie.coreos-bootstrap : Check if bootstrap is needed]     *******************************************************************************************************
```

Comment:
Can we shorten the output here, it's OK to mention that there will be output and the user only has to check for fatal errors.